edit/edit_test.go

```diff
@@ -93,6 +93,40 @@ func (test *whereTest) run(t *testing.T) {
 	}
 }
 
+func TestWhereLine(t *testing.T) {
+	tests := []whereLineTest{
+		{init: "Hello\n 世界!", addr: All, from: 1, to: 2},
+		{init: "Hello\n 世界!", addr: End, from: 2, to: 2},
+		{init: "Hello\n 世界!", addr: Line(1), from: 1, to: 1},
+		{init: "Hello\n 世界!", addr: Line(2), from: 2, to: 2},
+		{init: "Hello\n 世界!", addr: Regexp("/Hello"), from: 1, to: 1},
+		{init: "Hello\n 世界!", addr: Regexp("/世界"), from: 2, to: 2},
```

Comment:
Can we add some tests:
* init: ""
* The address spans multiple lines.
* A more than two lines.